src/CZ.Azure.FileExchange/CZ.Azure.FileExchange.csproj

```diff
@@ -1,16 +1,16 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="DartSassBuilder" Version="0.2.1-beta" />
+    <PackageReference Include="DartSassBuilder" Version="0.3.0-beta" />
```

Comment:
Should I try to remove this dependency?

Would be more complicated but could fix issues with the builds later on.

Reply:
yeah but maybe not in this PR. we should avoid making this PR to big.

Reply:
ok